example/template.env

```diff
@@ -1,55 +1,62 @@
-## Example / Template .env file for Wikibase release pipeline docker compose example
+# Example / Template .env file for Wikibase release pipeline docker compose example
 # WARNING: Do not add comments on the same line as env vars, as in some environments they will be included in the var!
 
-## Image Configuration
```

Comment:
These are moved to the bottom of the file as they are of top-line interest to us in updating this template, but of bottom-line interest to the user of this file.